pkg/legacy/config.go

```diff
@@ -22,19 +22,9 @@ import (
 // a v1 layer. See:
 // https://github.com/moby/moby/blob/master/image/spec/v1.md
 type LayerConfigFile struct {
```

Comment:
Maybe just `ConfigFile` instead of `LayerConfigFile`? Not sure on this.

Reply:
This struct is specifically used to only generated layer metadata which is why I kept the `Layer` in the name. The legacy package still uses `v1.ConfigFile` for image configs.